### PR TITLE
add pl-psychology to cosmos theme

### PR DESCRIPTION
**Pull request by ajluk**
Issue: #3538

### CODE REVIEW

File: styles/books/economics/book.scss [hunk 1]
```diff
@@ -9,7 +9,8 @@ $bandColor: #DCB83D;
 @import 'framework/framework';
 @import '../../design-settings/cosmos/_design.scss';
 @import '../../design-settings/cosmos/_settings.scss';
-@import '../../designs/cosmos/parts/pdf/_folio.scss';
+@import '../../designs/cosmos/pdf/folio';
+
```
Comment (wilkus27) [inline styles/books/economics/book.scss:13]: What is the reason for this change? I'm just curious :wink:

File: styles/designs/cosmos/pdf/locale/pl/_folio.scss [hunk 2]
```diff
@@ -158,7 +158,7 @@
     @include folio($prefaceContent, $prefaceContent, $accessContent, preface );
     @include folioWithBand($eocContent, $eocContent, $accessContent, eoc);
     @include folioWithBand($appendixContent, $appendixContent, $accessContent, appendix);
-    @include folioWithBand($eobContent, $eobContent, $accessContent eob);
+    @include folioWithBand($eobContent, $eobContent, $accessContent, eob);
```
Comment (wilkus27) [inline styles/designs/cosmos/pdf/locale/pl/_folio.scss:161]: Nice! You have good eye :relaxed: It's strange that it doesn't have impact on pl-u-physics :thinking: It shouldn't work without comma... But I don't see any changes in output files for physics. Interesting :female_detective:

Comment (wilkus27) [reply styles/designs/cosmos/pdf/locale/pl/_folio.scss:161]: Oh ok :facepalm: We have separate folio files for each theme, that's it doesn't have impact on physics :smile:

**Review by LostMojo — APPROVED** [review]

File: styles/books/economics/book.scss (hunk 1 above)
Comment (ajluk) [reply styles/books/economics/book.scss:13]: @wilkus27 I should have added this comment in the first place! 😅 The reason for this change is that I have created this file in the wrong catalog previously. In the rest of the themes this file is located directly in the `pdf` catalog, not in `parts`, so I fixed it for consistency.

**Review by wilkus27 — APPROVED** [review]
